OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -524,32 +595,64 @@ - (void)placeHolderImageViewDidDismissOnVideoView:(OTVideoView *)videoView {
 }
 
 - (void)setPublishAudio:(BOOL)publishAudio {
+    [[MultiPartyLoggingWrapper sharedInstance].logger logEventAction:KLogActionEnableLocalMedia
+                                                 variation:KLogVariationAttempt
+                                                completion:nil];
     if (!_publisher) return;
     _publisher.publishAudio = publishAudio;
+    [[MultiPartyLoggingWrapper sharedInstance].logger logEventAction:KLogActionEnableLocalMedia
+                                                 variation:KLogVariationSuccess
+                                                completion:nil];
```

Comment:
For cases like this, why do we even have `attempt` if we don't have a case to fail?